### PR TITLE
scripts and docs for mdm onboarding

### PR DESCRIPTION
**Pull request by ganga1980**
This PR contains following changes 

    -  Bash and PS scripts for  Monitoring Metrics Publisher role assignment for single  or all the aks clusters in specified subscription
  -  Instructions how to onboard the role assignment with Az CLI or PS

### CODE REVIEW

File: docs/aks/mdmonboarding/mdm_onboarding.ps1 [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+#
+#  Add Monitoring Metrics Publisher role assignment to the AKS cluster resource
+#
+
+New-AzRoleAssignment -ApplicationId $servicePrincipalClientId -scope $clusterResourceId -RoleDefinitionName "Monitoring Metrics Publisher"  -ErrorVariable assignmentError -ErrorAction SilentlyContinue
+
+if ($assignmentError) {
+
+    if($assignmentError.Exception -match "role assignment already exists") {
```
Comment (bragi92) [inline docs/aks/mdmonboarding/mdm_onboarding.ps1]: New-AzRoleAssignment always returns the text in English? checking the code might be better.

File: docs/aks/mdmonboarding/mdm_onboarding.ps1 [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    $choices.Add((New-Object Management.Automation.Host.ChoiceDescription -ArgumentList '&Continue without installing the Module'))
+    $choices.Add((New-Object Management.Automation.Host.ChoiceDescription -ArgumentList '&Quit'))
+
+    $decision = $Host.UI.PromptForChoice($message, $question, $choices, 0)
+
+    switch ($decision) {
+	    0 { 
+
+           try {
+			    Write-Host("Installing Az.Resources...")
```
Comment (bragi92) [inline docs/aks/mdmonboarding/mdm_onboarding.ps1]: A check here to install the module only if it is required.
if ($azAccountModule -eq $null) {
// install 
}

Comment (ganga1980) [reply docs/aks/mdmonboarding/mdm_onboarding.ps1]: done.

File: docs/aks/mdmonboarding/mdm_onboarding.ps1 (hunk 1 above)
Comment (ganga1980) [reply docs/aks/mdmonboarding/mdm_onboarding.ps1]: Not sure how the rest API handles localization. For now, I verified whether role assignment exists or not through GET call.

**Pull request merged**